openshift/templates/db/postgresql-build.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Does this make sense as an imagestream and not as a build config? it seems silly to rebuild the postgres image every time the pipeline runs. However, not including postgres in the pipeline means that the postgres db won't be deployed automatically when the pipeline runs in a new environment for the first time.

Reply:
It's sucked in as an image stream in order to have better control over the versions and to ensure it's managed consistently with the rest of the images and deployments.  It is not deployed automatically to `dev` so a conscious decision is required when deploying any Postgres version initially.  Once deployed to `dev` the image is managed by the deployment pipelines.